src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label for="searchBig">
+        <input
+          type="search"
+          name="searchBig"
+          id="searchBig"
+          class="input input--big"
```

Comment:
```suggestion
          class="input"
```
delete useless classes